core/trino-server-rpm/pom.xml

```diff
@@ -232,7 +258,7 @@
                                         </file>
                                         <file>
                                             <when>
-                                                <suffix>/usr/lib/trino/bin/launcher.py</suffix>
+                                                <suffix>/usr/lib/trino/bin/launcher</suffix>
```

Comment:
I assume this ensures that the startup command stays the same despite the filenames above being launcher2